scripts/docker-compose-build.sh

```diff
@@ -11,13 +11,13 @@ ARGS="--cache-from $DOCKER_REPOSITORY/unlock-core:master"
 docker build -t unlock-core -f $REPO_ROOT/docker/unlock-core.dockerfile $ARGS $REPO_ROOT
 
 # First build all of the images
-$REPO_ROOT/scripts/build-image.sh unlock-app true &
-$REPO_ROOT/scripts/build-image.sh wedlocks true &
-$REPO_ROOT/scripts/build-image.sh smart-contracts true &
-$REPO_ROOT/scripts/build-image.sh paywall true &
-$REPO_ROOT/scripts/build-image.sh locksmith true &
-$REPO_ROOT/scripts/build-image.sh unlock-protocol-com true &
-$REPO_ROOT/scripts/build-image.sh integration-tests true &
+. $REPO_ROOT/scripts/build-image.sh unlock-app true &
```

Comment:
note: this script was not passing environment variables to the child scripts, which caused them to rebuild every single time they run rather than pulling from cache locally. I don't think this change will affect remote CI, but it's possible that it might speed things up, we'll see.